src/components/pages/Landing/landingPageStyledComponents.js

```diff
@@ -34,35 +20,78 @@ export const ParagraphOneFormater = styled.div`
-It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
-It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with 
-desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.`,
+  `Explore our network of professional groomers! Our groomers are highly motivated and 
+  certified professionals eager to meet and serve your and your pets. Come find a Groomer near you today!`,
 ];
 
 export const SecondParagraphContent = [
-  `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque nisl eros, 
-pulvinar facilisis justo mollis, auctor consequat urna. Morbi a bibendum metus. 
-Donec scelerisque sollicitudin enim eu venenatis. Duis tincidunt laoreet ex, 
-in pretium orci vestibulum eget. Class aptent taciti sociosqu ad litora torquent
-per conubia nostra, per inceptos himenaeos. Duis pharetra luctus lacus ut 
-vestibulum. Maecenas ipsum lacus, lacinia quis posuere ut, pulvinar vitae dolor.
-Integer eu nibh at nisi ullamcorper sagittis id vel leo. Integer feugiat 
-faucibus libero, at maximus nisl suscipit posuere. Morbi nec enim nunc. 
-Phasellus bibendum turpis ut ipsum egestas, sed sollicitudin elit convallis. 
-Cras pharetra mi tristique sapien vestibulum lobortis. Nam eget bibendum metus, 
-non dictum mauris. Nulla at tellus sagittis, viverra est a, bibendum metus.`,
+  `Let our app make the process of booking appointments for your pets easy. Our applications features 
+  will streamline the processs of booking a groomer for your pet, and will send you text message updates
+  about the status of your pet and appointment. 
+  `,
+];
+
+export const FirstReviewBlurb = [
+  `Tim Rogers says: Wow! I've really liked the service I recieved from my Groomer Jake for my dog Bubbus. 
+Bubbus is a very small and temparmental dog, and doesn't take to others very easily, 
+but I was very pleased to see how quickly he took to Jake. Jake cleaned Bubbus up good 
+as new! I highly recommend him(Jake) and the app!`,
+];
+
+export const SecondReviewBlurb = [
+  `Jojo Jackson says: My cat Minnie is outside very often, and as working cat Mom I find it difficult to keep her coat as clean I know it can be. It's such a shame when she's dirty; she's a goregous dog!
+Anyway, I came across this app Pet-Xpress, and was very intrigued. I found the app incredibly user friendly and I was able to find someone to groom Minnie very quickly!
+5/5 stars, highly recommended!`,
+];
+
+export const ThirdReviewBlurb = [
+  `Alexander Sheffield says: I'm a busy guy with a great dog. I make time to play with Rick everyday, but after working all day, it's hard to
+always clean Rick as much as I would like to. Using Pet-Xpress I'm able to have someone come out and groom my dog so his coat is nice, and it's nice to have professionals
+a couple of button clicks away to be able to groom my dog.`,
 ];

```

Comment:
great job adding content to the landing page! this will help the project look like a real product